specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/README.md

```diff
@@ -4,25 +4,35 @@
 
  **_NOTE:_** New events must not be delivered to Event Grid production endpoints until the events have been reviewed with the Azure SDK Architecture board and the PR is merged into main. Once the PR is merged to main, the events are considered GA regardless of whether docs list them as being in preview. This is because customers cannot control which version of events they consume - it is entirely up to the service publishing the events. Any breaking changes to events would need to be implemented as a new event type. Full details can be found in the [Azure Breaking Changes Policy](http://aka.ms/AzBreakingChangesPolicy/), Section 4.
 
-In order to automate the mapping of event definition with event type, please follow the guidelines below when adding new events to your typespec:
-- The name of a new event definition should have `EventData` suffix. For e.g. `AcsChatMessageReceivedEventData`.
-- The description of the new event should include the event type. This is the `eventType` name in an `EventGridEvent` or `type` name in `CloudEvent`. For e.g. `"Schema of the Data property of an EventGridEvent for a Microsoft.Communication.ChatMessageReceived event."` Here `Microsoft.Communication.ChatMessageReceived` is the event name. If your event is in preview, you may add the word preview: `"Schema of the Data property of an EventGridEvent for a Microsoft.Communication.ChatMessageReceived preview event."`
-- Make sure to add add your service's SystemEvent CloudEvent Schema examples under `samples/stable/2024-01-01/YOUR_SERVICE_NAME/`.
-
-### Write in Typespec
+## Write in Typespec
 
 Under the `Azure.Messaging.EventGrid.SystemEvents` folder find or create your service's `.tsp` file. This is where you will add your new event. For help with typespec conventions refer to [this doc](https://microsoft.github.io/typespec/) about typespec basics. Each new event will be represented as a typespec `model`. After you create your new event, in the `client.tsp` file, you need to add `@@usage(EventGrid.YourEventName, Usage.output)` and `@@access(EventGrid.YourEventName, Access.public)`.
 
+### Versioning
+
+When an event or a new property is added to the typespec it requires a release version. This is the first day of the month, of the current year, that you want your event to be released in. 
+
+For example, `v2025-02-01`, February 1st 2025 is targeting a release in February. 
+
+To add this release version:
+- In the `main.tsp` in the `enum ServiceApiVersions` place your intended release version. For example, `v2025_02_01: "2025-02-01"`.
+- When adding new properties or events to the TypeSpec, use the [versioning](https://github.com/microsoft/typespec/blob/main/packages/versioning/README.md) decorators. The most significant one is `@added(YOUR_RELEASE_VERSION)` for models or properties that are new.
```

Comment:
There should be no reason to be anything else than `added`, since it will be a breaking change. Maybe phrase it in a more direct way, like "use the @added decorator this way".